README.rst

```diff
@@ -10,6 +10,14 @@ The Python interface to the Redis key-value store.
 .. image:: https://badge.fury.io/py/redis.svg
         :target: https://pypi.org/project/redis/
 
+This has been forked from https://github.com/andymccurdy/redis-py and is intended to be
+used against the Yedis API end point of YugaByteDB. This client also supports additional
+commands (such as the time-series related commands) for use againist YugaByte DB that
+are not part of the official Redis API.
+
+Please refer to `YugaByte DB docs <https://docs.yugabyte.com/>`_ for reference.
```

Comment:
Is there an extra _ after the url? If so, can be removed...

Reply:
Tried removing it. Looks like we need it for the anchor text to be seen properly. Adding it back.